docs/src/main/paradox/routing-dsl/directives/alphabetically.md

```diff
@@ -150,7 +150,6 @@
 |@ref[storeUploadedFiles](file-upload-directives/storeUploadedFiles.md)                                               | Streams all uploaded files from a multipart request to files on disk                                                                                                                                      |
 |@scala[@ref[textract](basic-directives/textract.md)]                                                                         | @scala[Extracts a number of values using a `RequestContext => Tuple` function]                                                                                                                                     |
 |@scala[@ref[tprovide](basic-directives/tprovide.md)]                                                                         | @scala[Injects a given tuple of values into a directive]                                                                                                                                                          |
-|@ref[uploadedFile](file-upload-directives/uploadedFile.md)                                                           | Streams one uploaded file from a multipart request to a file on disk                                                                                                                                      |
```

Comment:
What happened to this? Was it renamed and the page duplicated?

Reply:
It has been deprecated since 10.0.11, so I replaced the documentation page with a redirect to the replacement (https://github.com/akka/akka-http/pull/3114/files#diff-3cd40b5f95731f5d8b37e8cc9a01fc63R17).

I guess it's time to remove the code itself as well?

Reply:
Yes, removing sounds good.